services/core/VolttronCentral/README.md

```diff
@@ -58,3 +58,9 @@ below. in the following.
             }
         }
     }
+
+# Security Considerations
+
+When deploying an instance of VOLTTRON Central, it is important to consider security.
+Please refer to the documentation for secure-deployment-considerations. In particular, 
+it would be recommended to consider the use of a reverse proxy.
```

Comment:
we recommend. the volttron community recommends?  

Should the secure-deployment-consideration be a link?

Reply:
Yes? It is not set up as a link. I would like it to link, but not sure how to link this README.md to the rest of the documentation, aside from a hyperlink to the readthedocs page, which I don't think is ideal.

Reply:
Spoke with Shwetha. added link to readthedocs page (main branch)